msvc/mono-netcore.sln

```diff
@@ -0,0 +1,220 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I am not sure this is right location. We don't want to mirror this and I'd prefer netcore sln not to be mirrored at all and live in dotnet/runtime

Reply:
The current plan is to mirror the whole msvc/ directory so it's easiest if this file is here too. The sln file typically doesn't need changes so it shouldn't be an issue.

Reply:
I didn't see it in the repo consolidation plan and this actually goes against the prefered policy of mirroring the minimum files possible

Reply:
this is the minimum possible since we don't want to diverge the .vcxproj's (same story as the Makefiles)